1 Niagara 4 Overview Exercise - BFile/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -0,0 +1,240 @@
+package com.tridiumuniversity.myFile;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.status.BStatus;
+import javax.baja.status.BStatusNumeric;
+import javax.baja.status.BStatusString;
+import javax.baja.sys.Action;
+import javax.baja.sys.BAbsTime;
+import javax.baja.sys.BComponent;
+import javax.baja.sys.BMonth;
+import javax.baja.sys.Flags;
+import javax.baja.sys.Property;
+import javax.baja.sys.Slot;
+import javax.baja.sys.Sys;
+import javax.baja.sys.Type;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "fileName",
+        type = "BStatusString",
```

Comment:
It's not a big deal, but the assignment asked you to use a `String` for this instead of a `BStatusString`. Using a `BStatusString` is actually a little harder, so good job! You see `BStatusString` a lot when working with StringPoints and StringWritables in Niagara.